src/App.jsx

```diff
@@ -137,7 +137,7 @@ function App() {
 
           if (selectedGraphic.layer.title === ROADS_LAYER_NAME) {
             setSelectedRoadFeature(selectedGraphic);
-          } else {
+          } else if (selectedGraphic.layer.title === END_POINTS_LAYER_NAME) {
```

Comment:
**suggestion**: hittest can take a layer option so you can restrict it to a specific set of data.

<img width="900" alt="image" src="https://github.com/agrc/plpco-access-map/assets/325813/a65a5499-e0ff-4f65-a0f5-b5e41645c36a">

Reply:
I'm not sure how this would help. I still need to use if statements to set the correct state. I assume that Esri runs the same sort of if statements if you pass the include option.